docker_compose_swarm_mode.py

```diff
@@ -214,9 +214,23 @@ def start(self, services=None):
 
 
 def main():
+    for k, v in {
+        'COMPOSE_FILE': 'docker-compose.yml',
+        'COMPOSE_HTTP_TIMEOUT': '60',
+        'COMPOSE_TLS_VERSION': 'TLSv1'
+    }.iteritems():
+        if not k in os.environ:
+            os.environ[k] = v    
+
+    env_path = os.path.join(os.getcwd(), '.env')
+
+    if os.path.isfile(env_path):
+        with open(env_path) as env_file:
+            map(lambda e: os.environ.update({e[0]:e[1]}), (line.strip().split('=', 1) for line in env_file if not line.startswith('#') and line.strip()))
```

Comment:
Good call, I missed that in the docs! I'll limit the updates from .env to only if the key is not already in os.environ, and move the remaining defaults down.